modules/dnn/src/layers/resize_layer.cpp

```diff
@@ -45,10 +45,15 @@ class ResizeLayerImpl : public ResizeLayer
                          std::vector<MatShape> &outputs,
                          std::vector<MatShape> &internals) const CV_OVERRIDE
     {
-        CV_Assert_N(inputs.size() == 1, inputs[0].size() == 4);
+        CV_Assert_N(inputs.size() == 1 || inputs.size() == 2, inputs[0].size() == 4);
         outputs.resize(1, inputs[0]);
```

Comment:
Maybe let's move it under `if` but instead of
```
            outputs[0][2] = inputs[1][2];
            outputs[0][3] = inputs[1][3];
```
do
```
outputs.resize(1, inputs[1]);
```

Reply:
`output[0][1] = inputs[0][1] != inputs[1][1] `